client/my-sites/plans/current-plan/index.jsx

```diff
@@ -69,6 +77,21 @@ class CurrentPlan extends Component {
 				' its professional features.' );
 		}
 
+		if ( plan === PLAN_JETPACK_PERSONAL || plan === PLAN_JETPACK_PERSONAL_MONTHLY ) {
```

Comment:
Added taglines to plans constants in https://github.com/Automattic/wp-calypso/pull/17484/commits/b5ffd04ed4aca55bc06a3b1c98a5bfe6d15f62d4 and changed the current plan header to use them in https://github.com/Automattic/wp-calypso/pull/17484/commits/110ebd37f4688fc9413dac3a87122ecf83bbdb22.